stepper/stepper_controller/include/L6470_pkg/L6470_lib.py

```diff
@@ -0,0 +1,563 @@
+    # ==============================================================================================
+    # Send a single command byte without an expected response.
+    def sendCmd(self, command):
+        # Send command byte to the controller.
+        byte = [command]
+        self._SPI.xfer2(byte)
+
+    # Send a command with the parameter split in 3 bytes and obtain the response in 3 bytes.
+    def sendCmd3(self, command, param):
+        # Split in 3 bytes, from MSB to LSB.
+        param2 = (param >> 16) & 0xFF
+        param1 = (param >> 8) & 0xFF
+        param0 = param & 0xFF
+        
+        # Send bytes to the controller separately and build response value.
+        # The bytes have to be sent separately since the Chip Select has to be reset
+        # between every byte.
+        request = [command]
+        response = self._SPI.xfer2(request)
+        responseValue = 0x000000
+        
+        request = [param2]
+        response = self._SPI.xfer2(request)
+        responseValue = responseValue | (response[0] << 16)
+        
+        request = [param1]
+        response = self._SPI.xfer2(request)
+        responseValue = responseValue | (response[0] << 8)
+        
+        request = [param0]
+        response = self._SPI.xfer2(request)
+        responseValue = responseValue | response[0]
+        
+        return responseValue
+
+    # Convert an acceleration from step/tick^2 to step/s^2.
+    def accToStepSec(self, StepTick2):
+        # Controller gives an acceleration in step/tick^2 formatted in unsigned fixed point 0.40.
+        # Equation: StepSec2 = (StepTick2 * 2^-40)/TICK^2
+        # Equivalent to: StepTick2 / 0.068719
+        StepSec2 = StepTick2 / self.STEP_ACC_RATIO
+        
+        return StepSec2
+
+    # Convert an acceleration from step/s^2 to step/tick^2.
+    def accToStepTick(self, StepSec2):
+        # Controller expects an acceleration in step/tick^2 formatted in unsigned fixed point 0.40.
+        # Equation: StepTick2 = (StepSec2 * TICK^2)/2^-40
+        # Equivalent to: StepTick2 * 0.068719
+        StepTick2 = int(StepSec2 * self.STEP_ACC_RATIO)
+        
+        # Limit speed to a 12 bits number
+        if StepTick2 > 0x00000FFF:
+            return 0x00000FFF
+        
+        return StepTick2
+
+    # Convert a speed from step/tick to step/s.
+    def speedToStepSec(self, StepTick):
+        # Controller gives a speed in step/tick formatted in unsigned fixed point 0.28.
+        # Equation: StepSec = (StepTick * 2^-28)/TICK
+        # Equivalent to: StepTick / 67.108864
+        StepSec = StepTick / self.STEP_SPEED_RATIO
+        
+        return StepSec
+
+    # Convert a speed from step/s to step/tick.
+    def speedToStepTick(self, StepSec):
+        # Controller expects speed in step/tick formatted in unsigned fixed point 0.28.
+        # Equation: StepTick = (StepSec * TICK)/(2^-28)
+        # Equivalent to: StepSec * 67.108864
+        StepTick = int(StepSec * self.STEP_SPEED_RATIO)
```

Comment:
int() is the equivalent for floor(). If precision is important here we should probably use math.round()